packages/api-contract/src/Abi/index.ts

```diff
@@ -53,7 +53,23 @@ function getLatestMeta (registry: Registry, json: Record<string, unknown>): Cont
   return converter[1](registry, metadata[`as${converter[0]}`]);
 }
 
-function parseJson (json: Record<string, unknown>, chainProperties?: ChainProperties): [Record<string, unknown>, Registry, ContractMetadataLatest, ContractProjectInfo] {
+function getEnvTypes (env: ContractEnvironment | undefined, lookup: PortableRegistry) {
```

Comment:
This should be defined, with default values? (Since it is part of the latest definition).

Since it is there, it would mean that the type check below should not be needed.